CHANGES.txt

```diff
@@ -46,6 +46,7 @@ RELEASE  VERSION/DATE TO BE FILLED IN LATER
       Calls now look like 'debug("template %s", text)' rather than
       'debug("template %s" % text)' so the logging system does the
       interpolation only when/if needed (was a pylint warning).
+    - Update Help display a bit - join some lines, drop "ignored for compat".
```

Comment:
Perhaps add
(help output with command line option `-H`)

Reply:
I updated this line (there was also a new change, which was added to this entry)